### PR TITLE
english cpp code style sheet added

### PR DESCRIPTION
**Pull request by attiladoor**

### CODE REVIEW

File: C++.en.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+Wiren Board C++ Style Guide
+========================
+
+The following sheet is based on Yandex code style for C++. If any guideline is missing from the following document, 
+please follow Yandex official documentation: https://clickhouse.yandex/docs/en/development/style.html . But if you
```
Comment (evgeny-boger) [inline C++.en.md]: looks like this document is made by another team and thus different...
The one our conventions were based at is called Arcadia C++ Style Guide, and I'm afraid it's not public

File: C++.en.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+see any collision between documents, please follow the guide below. 
+
+## Marking
+
+For identation blocks only spaces are used and 4 spaces count as 1 tab.
```
Comment (evgeny-boger) [inline C++.en.md]: "as 1 indentation level"

File: C++.en.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+Aritmetic operations are always separated by spaces:
+```C++
+int sum = a + b;
+```
+Whitspace should be placed before each bracket opening.
+```C++
+int function (int a, int b, int c) 
+{
+  int d = abs ( (a * sqrt (b)) + c);
```
Comment (evgeny-boger) [inline C++.en.md]: the example already breaks the rule above :) Better remove the rule at all.

File: C++.en.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+}
+```
+
+At sequence of variables comma should be placed right after variable name and the comma should be followed by space before
+next variable name:
+
+```C++
+int sum = GetSum (a, b);
+```
+
```
Comment (evgeny-boger) [inline C++.en.md]: the same goes for every punctuation mark, whether it's comma or semicolon or colon or whatever else

File: C++.en.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+zipNumbers[i] = zipNumbers[j];
+zipNumbers[j] = buffer;
+```
+
+Short names, like *i ,j* are basically allowed only as loop variables. 
+At shortened names, variable name should be started with capital letter: *TMqttClient*
+
+### Classes
+
+Class names should be started with capital letters:  *TModbusClient*.
```
Comment (evgeny-boger) [inline C++.en.md]: moreover, it should start with capital T

File: C++.en.md (hunk 1 above)
Comment (evgeny-boger) [reply C++.en.md]: on a second thought, it's very closely related to that Arcadia one, so please keep it

File: C++.en.md (hunk 3 above)
Comment (attiladoor) [reply C++.en.md]: Ok

File: C++.en.md (hunk 4 above)
Comment (attiladoor) [reply C++.en.md]: Are you sure it should be used for all? It would result something like this:
if (! variable) { }
instead of:
If ( !variable) { }

**Pull request merged**